### PR TITLE
Improve operator contract docs and ergonomics

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: contracts/src/v0.7/ChainlinkClient.sol [hunk 1]
```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.7.0;
 import "./Chainlink.sol";
```
Comment (alexroan) [inline contracts/src/v0.7/ChainlinkClient.sol:4]: What's the plan for porting these changes backwards and forwards?

File: contracts/src/v0.7/ChainlinkClient.sol [hunk 2]
```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.7.0;
...
 import "./interfaces/ENSInterface.sol";
```
Comment (alexroan) [inline contracts/src/v0.7/ChainlinkClient.sol:5]: Can we upgrade the contracts changelog with this change?

File: contracts/src/v0.7/Operator.sol [hunk 3]
```diff
@@ -39,13 +39,14 @@ contract Operator is
   // We initialize fields to 1 instead of 0 so that the first invocation
```
Comment (alexroan) [inline contracts/src/v0.7/Operator.sol:39]: Contracts changelog entry needed for this.

**Review by alexroan — CHANGES_REQUESTED** [review]

File: contracts/src/v0.7/ChainlinkClient.sol (hunk 1 above)
Comment (se3000) [reply contracts/src/v0.7/ChainlinkClient.sol:4]: Get the reference version in and then copy the changes over in a separate PR.

File: contracts/src/v0.7/Operator.sol (hunk 3 above)
Comment (alexroan) [reply contracts/src/v0.7/Operator.sol:39]: Can you add a reference in the changelog to the fact that Operator and ChainlinkClient in v0.7 has been updated with these changes, but not v0.6 or v0.8 yet?

File: contracts/src/v0.7/ChainlinkClient.sol (hunk 2 above)
Comment (se3000) [reply contracts/src/v0.7/ChainlinkClient.sol:5]: Noted.

File: contracts/src/v0.7/Operator.sol (hunk 3 above)
Comment (se3000) [reply contracts/src/v0.7/Operator.sol:39]: Updated.

**Review by alexroan — APPROVED** [review]

**Pull request merged**